src/content/energy_consumption/index.md

```diff
@@ -0,0 +1,78 @@
+
+Ethereum has implemented the PoW protocol since its inception. While migration away from PoW has always been a fundamental goal of Ethereum, it has also arguably been the most philosophically and technologically challenging because the viable alternatives all required some compromise in one of Ethereum's core principles.
+
+## PoW energy expenditure
+
+PoW is a very robust way to secure the against dishonest changes to the blockchain, but it is problematic for several reasons. Since the right to mine a block requires solving a computational puzzle, miners can increase their odds of success by investing in more powerful hardware, leading to an arms race of increasingly expensive and power-hungry mining equipment. Over time, this prices miners out of the market, tending towards centralisation, and also dramatically increases the energy consumption associated with securing the network. Ethereum's PoW protocol currently consumes as much energy as a medium-sized country.
+
+## PoS
+
+A greener future for Ethereum is already being built in the form of a proof-of-stake (PoS) chain. Under PoS, the arbitrary puzzle solving is not necessary, drastically reducing the energy expenditure required to secure the network. Miners are replaced by validators who perform the same function except that instead of expending their assets up-front in the form of computational work, they stake ETH as collateral against dishonest behaviour. If the validator's node is non-responsive, or a fraudulent block is submitted to the chain, the staked assets can be "slashed", strongly incentivizing honesty and securing the network. Similarly to PoW, in order to maintain a fraudulent blockchain, a validator would require 51% of the total ETH staked in the network. Unlike PoW, however, consensus is not based on the longest chain, but on a mechanism known as ["casper"](https://arxiv.org/abs/1710.09437). Migrating from PoW to PoS eliminates the need to expend energy on arbitrary computations.
+
+## The merge
+
+There is now a functional PoS blockchain called the [Beacon Chain]("https://ethereum.org/en/eth2/beacon-chain/") that has been running since December 2020 that is demonstrating the viability of the PoS protocol. Now that the ["London Upgrade"](https://ethereum.org/en/history/#london) has been successfully implemented, the switch from PoW to Pos (known as "the merge") is the next major update coming to Ethereum with a target date of late 2021 - early 2022. The merge to PoS will also bring associated efficiency gains through "sharding" the blockchain into sub-chains and accelerating the adoption of "roll-ups" (collections of transactions settled cheaply off-chain then pushed to the Ethereum blockchain as a single batch transaction).
```

Comment:
> as been successfully implemented, the switch from PoW to Pos (known as "the merge") is the next major update coming to Ethereum with a target date of late 2021 - early 2022.

Altair is before the Merge

Reply:
> The merge to PoS will also bring associated efficiency gains through "sharding" the blockchain into sub-chains

The merge will happen before shard chains now

Reply:
> accelerating the adoption of "roll-ups"

Not sure what you mean by this. I don't see the merge and roll ups as being linked?